packages/babel-plugin-remove-graphql-queries/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -54,6 +54,17 @@ export default (() => React.createElement(StaticQuery, {
 }));"
 `;
 
+exports[`Transforms queries defined in own variable in useStaticQuery 1`] = `
+"import staticQueryData from \\"public/static/d/2626356014.json\\";
+import React from 'react';
+import { useStaticQuery } from 'gatsby';
```

Comment:
Should remove this `import` in the babel transform

Reply:
This is a micro optimization and done now

But it only works for instances of `import` and not `require` or object accessors on a top level require of gatsby

I think it's okay to mark this as resolved and fix later since it won't break anything but I'd love to hear what you have to say @pieh @DSchau